### DISCUSSION TITLE
[Enhancement] [2.0] Add commented-out examples to Custom CSS page

### DISCUSSION THREAD

**Discussion by Redsandro** [Zettlr/Zettlr#2398] [Ideas]
### Description

Tempt people to tinker with custom CSS.

### Proposed Changes

Apart from adding a keyboard shortcut (#2379) and mentioning it on the themes page (#2380), by default, upon opening the CSS editor for the first time, there could be some basic examples (commented out of course) so the user is tempted to uncomment them and alter them.

---

Backlinks: #2243

**Comment by nathanlesage** [comment]
Interesting idea, but I think that should be a discussion, since the actual change then would just incorporate copying and pasting the successful examples over to the Custom CSS page!

Moving …

**Reply by cococow123** [reply]
As someone who is not entirely sure what visual changes I would make to Zettlr, I think maybe a link to a page full of examples would be good, similar to the Custom CSS page. The only change I currently need to figure out is how to make the checked tasklist boxes match the primary green color.

**Reply by nathanlesage** [reply]
> The only change I currently need to figure out is how to make the checked tasklist boxes match the primary green color.

That's actually quite difficult, since for that you'd need a completely custom element, and that's nasty … unfortunately, due to them being Vue-components, I can't really push them into the editor area :(
